libs/explorers/shared/src/lib/components/wiki-hero/wiki-hero.component.scss

```diff
@@ -4,7 +4,7 @@
 #wiki-hero-container {
   display: flex;
   flex-direction: column;
-  min-height: calc(100vh - var(--header-height) - var(--footer-height) + 1px);
+  min-height: calc(100vh - var(--header-height) - var(--footer-height));
```

Comment:
This 1px offset (relic from Agora can now be removed).  This is responsible for the vertical scrollbars.